dev/protos/update.sh

```diff
@@ -89,6 +89,7 @@ PBJS_ARGS=( --proto_path=. \
       
 "${PBJS}" "${PBJS_ARGS[@]}" -o firestore_v1_proto_api.js \
   "${PROTOS_DIR}/google/firestore/v1/*.proto" \
+  "${PROTOS_DIR}/google/firestore/proto/*.proto" \
```

Comment:
Is it possible to export this to a separate JS file as well? I have never tried this option, but I wonder if `--sparse` will allow you to reference external types (https://www.npmjs.com/package/protobufjs#pbjs-for-javascript) and not include the `v1` types twice.

Reply:
I tried `--sparse` and it simply throws error.